libcontainer/intelrdt/intelrdt.go

```diff
@@ -548,26 +556,47 @@ func GetIntelRdtPath(id string) (string, error) {
 	return path, nil
 }
 
+// Get the 'container_id" path in Intel RDT "monitoring group" filesystem.
+func GetMonGroupIntelRdtPath(id string) (string, error) {
```

Comment:
Are you going to use this function outside of the package? If not, why is it public?